src/handlers/comment/handlers/payout.ts

```diff
@@ -27,7 +27,7 @@ export const payout = async (body: string) => {
   }
 
   const IssueComments = await getAllIssueComments(issue.number);
-  if (IssueComments.length === 0) {
+  if (IssueComments.length == 0) {
```

Comment:
Why

Reply:
same thing. no need to use `==` instead of `===`